braid/access.c

```diff
@@ -68,7 +68,7 @@ _braid_FAccess(braid_Core     core,
 
          if (access_level >= 1)
          {
-            _braid_AccessStatusInit( ta[fi-ilower], fi, rnorm, iter, level, nrefine, gupper,
+            _braid_AccessStatusInit( ta[fi-ilower], fi-1, rnorm, iter, level, nrefine, gupper,
```

Comment:
@jbschroder Has this been a bug all along?  Can you explain what's behind this change.  Thanks!

Reply:
@rfalgout Good question!  I found that StepStatus always returns the time-index minus one, but that AccessStatus just returned the time-index.  So, I modified AccessStatus to be the same as StepStatus.  

Overall, I don't quite like the minus one idea, but it has gotten woven into things a bit with StepStatus (and possibly in user codes, too), so I hesitated to change that.  We could put this in the issue tracker as something to resolve.  Let me know what you think!

Reply:
Isn't there a bigger risk of impacting user code if we change to -1?  Users already call Access, correct?  I agree that we might want to discuss via an issue.  I sort of think that the current behavior makes sense, though.  Basically, status corresponds to the "current" time point, and in the context of `Step` that is `fi-1` because it takes that value and moves it forward.  In the context of `Access` the "current" time point is `fi`.

Reply:
I was thinking that user code for Access is mostly for outputting things to the screen or file, so the chance for impact would be small.  But I see your point about the time-index.  Inside of Step, tindex is basically the index of "tstart", but inside of Access, we can think of tindex as just the current time-point.  Is that correct?

If we want to keep it that way, I can roll back this change to AccessStatus and make a note in the comments/docs.  Let me know what you think.

Reply:
Correct.  The term "current" clarifies both situations, I think, because in Step point `fi` is the "next" time point (aligned with `tstop`).  I think we should roll these back.

Reply:
Sounds good.  I just pushed some changes to address this.  One issue is that when accessing the Richardson error estimate array, Step() needs it's time index shifted by one, whereas Access does not.  This led to the creation of two different StatusGetSingleErrorEst functions, one for Step and one for Access.  But, this was the best solution that I saw.

Reply:
Looks good.  Thanks!